sqlpage/templates/big_number.handlebars

```diff
@@ -13,19 +13,21 @@
         
         {{#if title}}
         <div class="d-flex align-items-center">
-          <!-- Otsikko (title): mahdollisuus linkkiin ja targetiin -->
           <div class="subheader text-truncate me-2">
+            {{!--  Introduced `title_link_new_tab` and `value_link_new_tab`.  If set to a truthy value, opens the link in a new tab (target="_blank").  Otherwise, defaults to same tab (target="_self"). --}}
```

Comment:
now I understand the comment, but I think we can do without it :) We have a human-readable changelog in /CHANGELOG.md 

```suggestion
```